src/utils/validatePhoneNumbers.tsx

```diff
@@ -38,8 +39,13 @@ export const fullPhoneNumberValidator = (fullNumber: string): boolean => {
   try {
     const phoneNumberUtil = new PhoneNumberUtil();
     const parsedNumber = phoneNumberUtil.parse(fullNumber);
+    const nationalNumber = parsedNumber.getNationalNumber(); // always return numbers without whitespace
     const regionCode = phoneNumberUtil.getRegionCodeForNumber(parsedNumber);
-    return phoneNumberUtil.isValidNumberForRegion(parsedNumber, regionCode);
+    if (regionCode === "SG") {
+      return DEFAULT_SG_PHONE_NUMBER_REGEX.test(`${nationalNumber}`);
```

Comment:
issit can just do this?

```typescript
 return DEFAULT_SG_PHONE_NUMBER_REGEX.test(nationalNumber);
```

Reply:
or this is to force it to become a string?

Reply:
to force it to become a string!